### PR TITLE
[KYUUBI #728] Support SQL State: 0A000 - feature_not_supported

### PR DESCRIPTION
**Pull request by zhaomin1423**
### _Why are the changes needed?_

Follow #670 to:
Apply errorCode 0A000: feature_not_supported to val errStatus = KyuubiSQLException("Feature is not  @available").toTStatus

### _How was this patch tested?_
- [ ] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [ ] Add screenshots for manual tests if appropriate

- [ ] [Run test](https://kyuubi.readthedocs.io/en/latest/tools/testing.html#running-tests) locally before make a pull request

### CODE REVIEW

**Comment by yaooqinn** [comment]
we shall call this at FrontendService for。unsupported features

File: kyuubi-common/src/main/scala/org/apache/kyuubi/KyuubiSQLException.scala [hunk 1]
```diff
@@ -81,6 +81,10 @@ object KyuubiSQLException {
     }
   }
 
+  def featureNotSupported(): KyuubiSQLException = {
+    KyuubiSQLException("feature not supported", sqlState = "0A000")
+  }
```
Comment (turboFei) [inline kyuubi-common/src/main/scala/org/apache/kyuubi/KyuubiSQLException.scala:86]: could you invoke it in the FrontendService for unsupported operation?

Comment (zhaomin1423) [reply kyuubi-common/src/main/scala/org/apache/kyuubi/KyuubiSQLException.scala:86]: done

**Review by yaooqinn — APPROVED** [review]

**Comment by ulysses-you** [comment]
thanks, merging to master for v1.3.0
